recipe/meta.yaml

```
@@ -10,15 +10,17 @@ source:
 
 build:
   number: 0
-  skip: true  # [osx or win]
+  skip: true  # [osx]
 
 requirements:
   run:
     - cuda-cupti-dev 12.0.90
-    - cuda-gdb 12.0.90
+    # cuda-gdb should be supported on ppc64le, but is currently broken
+    # ref https://github.com/conda-forge/cuda-gdb-feedstock/pull/4
+    - cuda-gdb 12.0.90       # [linux64 or aarch64]
```

Filed an issue on this feedstock to track adding this back on `ppc64le` ( https://github.com/conda-forge/cuda-command-line-tools-feedstock/issues/6 ) and have suggested including this below

Tweaked the selector logic slightly to make it clearly that `ppc64le` is the only thing being excluded

```suggestion
    # There is no `cuda-gdb` support on Windows so it is skipped.
    # Also `cuda-gdb` should be supported on `ppc64le`, but is currently broken.
    # xref: https://github.com/conda-forge/cuda-command-line-tools-feedstock/issues/6
    - cuda-gdb 12.0.90       # [linux and not ppc64le]
```